packages/scripts/scripts/test-e2e.js

```diff
@@ -27,6 +28,14 @@ const {
 	hasJestConfig,
 } = require( '../utils' );
 
+const result = spawn( 'node', [ 'node_modules/puppeteer/install.js' ], {
```

Comment:
I tested this with the following setup:

1. in wp-calypso root package.json, I changed @wordpress/scripts to a file ref (pointing at `gutenberg/packages/scripts`):

https://github.com/Automattic/wp-calypso/blob/2c726694096b9e9df1000dbe7e4e1e852b80682b/package.json#L186

2. I deleted both gutenberg and wp-calypso node modules. I had this branch checked out in Gutenberg. Then I ran npm i for both gutenberg and wp-calypso. I verified that I did not see a chromium install step in either repo :)
3. I tried to run e2e tests for our plugin (which is a monorepo app) ([see this PR](https://github.com/Automattic/wp-calypso/pull/39327)), but the tests failed with the following error: 

```sh
lerna info Executing command in 1 package: "npm run test:e2e"
> @automattic/full-site-editing@0.19.0 test:e2e /Users/noah.allen/source/wp-calypso/apps/full-site-editing
> npx wp-scripts test-e2e --wordpress-base-url='http://localhost:4013'
internal/modules/cjs/loader.js:796
    throw err;
    ^
Error: Cannot find module '/Users/noah.allen/source/wp-calypso/apps/full-site-editing/node_modules/puppeteer/install.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

It looks like it was referencing node_modules of the monorepo app, rather than the node modules of the root package.json (where @wordpress/scripts would be installed). I also don't see a `puppeteer` directory in the root node_modules of wp-calypso (maybe because of the file ref?), but I'm concerned that this might not work for plugins that use @wordpress/scripts as a dependency. :)

Reply:
Might be more reliable as... ?

```suggestion
const result = spawn( 'node', [ require.resolve( 'puppeteer/install' ) ], {
```

https://nodejs.org/api/modules.html#modules_require_resolve_request_options

Reply:
Nice, that does seem to fix it!! It looks as if chromium successfully downloads with this change.

However, I then get the following error:
```
> npx wp-scripts test-e2e --wordpress-base-url='http://localhost:4013'
Chromium downloaded to /Users/noah.allen/source/gutenberg/packages/scripts/node_modules/puppeteer/.local-chromium/mac-722234
● Validation Error:
  Preset jest-puppeteer is invalid:
  The "id" argument must be of type string. Received type object
  TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string. Received type object
    at validateString (internal/validators.js:112:11)
    at Module.require (internal/modules/cjs/loader.js:841:3)
    at require (internal/modules/cjs/helpers.js:74:18)
    at setupPreset (/Users/noah.allen/source/gutenberg/packages/scripts/node_modules/jest-config/build/normalize.js:230:14)
    at normalize (/Users/noah.allen/source/gutenberg/packages/scripts/node_modules/jest-config/build/normalize.js:554:15)
    at readConfig (/Users/noah.allen/source/gutenberg/packages/scripts/node_modules/jest-config/build/index.js:163:46)
    at readConfigs (/Users/noah.allen/source/gutenberg/packages/scripts/node_modules/jest-config/build/index.js:373:26)
    at /Users/noah.allen/source/gutenberg/packages/scripts/node_modules/@jest/core/build/cli/index.js:155:58
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/noah.allen/source/gutenberg/packages/scripts/node_modules/@jest/core/build/cli/index.js:108:24)
  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

I'm using the e2e script right out of Gutenberg without specifying any extra configuration.

Reply:
Thanks, @aduth for d080809. It should help when using outside of Gutenberg for sure 👍 

@noahtallen, thanks for testing and raising the issue in the first place.

Reply:
> Nice, that does seem to fix it!! It looks as if chromium successfully downloads with this change. (though it does not indicate that it's happening, it does say that it successfully downloads after everything finishes)

This is how it looks like on my end:

![wp-scripts-chromium](https://user-images.githubusercontent.com/699132/74506397-ca88af00-4ef9-11ea-9bff-5be28567855a.gif)

Reply:
👍 Sure thing!